client/navigation/action-creators.js

```diff
@@ -12,7 +12,7 @@ export function goToIndex(transitionFn = push) {
     if (lobby.inLobby && IS_ELECTRON) {
       transitionFn(`/lobbies/${encodeURIComponent(lobby.info.name)}`)
     } else if (channels.size) {
-      transitionFn(`/chat/${encodeURIComponent(channels.first())}`)
+      transitionFn(`/chat/${encodeURIComponent([...channels][0])}`)
```

Comment:
Not sure what the intention here is, but I'm pretty sure this is just an array and creating a new array just to grab the first value and throw it away seems pretty unnecessary?

Reply:
This is actually a `Set` in the reducer, but now that I think about it, this should probably be an `OrderedSet`, since getting the "first" element from it doesn't make much sense otherwise. We could also make the list of channels that the user has joined an `array`, but then we lose the uniqueness guarantee. Though I guess if it ever happens that we try to insert the same channel into the array twice, then something else is broken  ^^